roles/KubevirtCommonTemplatesBundle/templates/kubevirt-os-images-viewrole.yaml.j2

```diff
@@ -8,5 +8,5 @@ rules:
   resources: ["persistentvolumeclaims", "persistentvolumeclaims/status"]
   verbs: ["get", "list", "watch"]
 - apiGroups: ["cdi.kubevirt.io"]
-  resources: ["datavolumes/source"]
+  resources: ["datavolumes", "datavolumes/source"]
```

Comment:
this role is fine.  It is the rolebinding that needs to be updated to include:
```
subjects:
- kind: Group
  name: system:serviceaccounts
  apiGroup: rbac.authorization.k8s.io
```

That in addition to https://github.com/kubevirt/kubevirt/pull/4269 should result in a working solution

Reply:
You DO NOT want anyone to be able to create datavolumes in the golden namespace.

Were you testing with the most recent kubevirt master code?

Reply:
@mhenriks Our CI is not testing with master kubevirt, it is testing with the latest release, do you know if the code changes you made made it to a release?

Reply:
@omeryahud no

Reply:
Also `oc auth can-i create datavolumes/source --as system:serviceaccount:default:default -n kubevirt-os-images`

Should be

`oc auth can-i create datavolumes --subresource source --as system:serviceaccount:default:default -n kubevirt-os-images`

Reply:
Removed datavolumes